### ISSUE TITLE
Ruby ships with racc, why depend on it as gem?

### ISSUE THREAD

**Issue by luislavena** [codegram/gherkin-ruby#14]
Hello,

I recently noticed that for installing gherkin-ruby, racc gem dependency needs to be built, which defeats the purpose of having a pure ruby version.

The thing is that racc is already part of Ruby:

```
$ ruby -v
ruby 1.9.3p194 (2012-04-20 revision 35410) [x86_64-darwin10.8.0]

$ gem list racc

*** LOCAL GEMS ***

$ ruby --disable-gems -rracc/parser.rb -e "puts 'works'"
works
```

Perhaps the racc dependency can be move to development one instead?

Thank you! :heart:

**Comment by luislavena** [comment]
Forgot to mention, if you say is good, I can provide a pull request :wink:

**Comment by oriolgual** [comment]
Sounds good to me, I'd love a PR :D

Thanks Luis!

**Comment by josepjaume** [comment]
:heart: :heart:

**Comment by josepjaume** [comment]
Fixed in f68cd0279b932f8b6ac0917108c2776ca98433bc, thanks! :)

**Comment by txus** [comment]
Thanks :)
